next-packages/serialize/src/index.js

```diff
@@ -193,11 +193,10 @@ function createStringFromObject(
           )
         }
         if (
-          Array.isArray(obj[key]) &&
-          (typeof obj[key][0] === 'string' &&
-            registered[obj[key][0]] === undefined)
+          Array.isArray(value) &&
+          (typeof value[0] === 'string' && registered[value[0]] === undefined)
         ) {
-          obj[key].forEach(value => {
+          value.forEach(value => {
```

Comment:
wouldnt this be faster as regular for loop? or is perf diff negligible nowadays?

Reply:
Probably negligible but I'm gonna change it to a for loop anyway.

Reply:
if u do so u can use `let` for a `key` again :p